### PR TITLE
Add a contributing file

### LINKED ISSUE #26: create a CONTRIBUTING.md
document the steps necessary from cloning the repo, to installing dev dependencies, to running the test suite to verify new changes did not cause breakage, to making a PR and having the continuous integration tests run.

### PR DESCRIPTION
**Pull request by marwahaha**
closes #26

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Contributing
+
+🏥 We're glad you want to help!
+
+## Goals of the project
+
+*  *Stay lightweight*: depend on as much of the standard library as possible; minimize external dependencies.
```
Comment (jpurviance) [inline CONTRIBUTING.md]: can we fix this formatting.

**Review by jpurviance — CHANGES_REQUESTED** [review]
@ivanov you might know best on this. can you provide comment.

File: CONTRIBUTING.md (hunk 1 above)
Comment (scopatz) [reply CONTRIBUTING.md]: Curious as to what is wrong with this formatting?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+## Get the code
+
+1. Fork this repository.
+2. `git clone` the forked repository, and `cd` to it.
+3. Get the development libraries with `pip install -r requirements-dev.txt`
+
+## Testing your changes
+
+Just run `tox`.
```
Comment (ivanov) [inline CONTRIBUTING.md]: "Just" ends up being a barrier for folks who have no idea what `tox` is. Can we add a tox link, and remove the "just"?

**Review by ivanov — APPROVED** [review]
Thanks for contributing this (how meta! :D ) "Just" a small suggestion on my end and this looks good.

File: CONTRIBUTING.md (hunk 1 above)
Comment (marwahaha) [reply CONTRIBUTING.md]: Talked in person - There are two whitespace characters on this line, unlike the other bullet points.

**Review by jpurviance — APPROVED** [review]

**Pull request merged**